### PR TITLE
Add German Locale support

### PR DESCRIPTION
**Pull request by burghoffdavid**
Hi, I've added German locale support for GitBook.

Link to Diskussion: https://github.com/orgs/GitbookIO/discussions/537

### CODE REVIEW

**Comment by burghoffdavid** [comment]
@addisonschultz

File: src/intl/translations/de.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    cookies_title: 'Cookies',
+    cookies_prompt:
+        'Diese Website verwendet Cookies, um ihre Dienste bereitzustellen und den Datenverkehr zu analysieren. Durch die Nutzung dieser Website akzeptieren Sie die ${1}.',
+    cookies_prompt_privacy: 'Datenschutzrichtlinie',
+    cookies_accept: 'Akzeptieren',
+    cookies_reject: 'Ablehnen',
+    cookies_close: 'Schließen',
+    edit_on_git: 'Bearbeiten auf ${1}',
+    notfound_title: 'Seite nicht gefunden',
+    notfound: 'Die Seite, die Sie suchen, existiert nicht.',
```
Comment (thebiglabasky) [inline src/intl/translations/de.ts:40]: I was wondering about a simpler one like "Die gesuchte Seite existiert nicht"?

**Review by thebiglabasky — APPROVED** [review]
Looks good, thanks!
I made a comment but don't trust my German enough to consider that a must :)

File: src/intl/translations/de.ts (hunk 1 above)
Comment (burghoffdavid) [reply src/intl/translations/de.ts:40]: Good suggestion, changed it!

**Comment by burghoffdavid** [comment]
I just noticed that I did not import the new locale file inside `src/intl/translations/index.ts` to add it to the languages var. Should this be added with this PR or will it be added further down the line by the team?

**Comment by addisonschultz** [comment]
@burghoffdavid Could you please add the translation import in? Thank you!

**Comment by burghoffdavid** [comment]
@addisonschultz Done :heavy_check_mark:

**Comment by burghoffdavid** [comment]
@addisonschultz Not entirely sure why the typecheck CI is failing, any ideas?

**Comment by addisonschultz** [comment]
@burghoffdavid yep - I've got to add these types to our backend schema - working on that now 😄

**Comment by addisonschultz** [comment]
Hi @burghoffdavid - first off, I want to apologize for the delays on getting this PR up and live so slowly - it's brought to my attention a lot of workflow improvements we need to make on our end in order to make this a lot more smooth. 

Secondly, I wanted to let you know that your contribution is now live in GitBook!

Screenshot 2024-06-19 at 19 51 27

You can now update the locale to German - let me know if you have any questions!

**Pull request merged**